examples/gnrc_minimal/Makefile

```diff
@@ -13,7 +13,7 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
 #USEMODULE += xbee
 
 ## set default UART to use in case none was defined
-#XBEE_UART ?= "UART_NUMOF-1"
+#XBEE_UART ?= "1"
```

Comment:
With `CFLAGS += -DDEVELHELP` it crashes with a failed assertion in both conditions.